packages/app/src/cli/services/dev/processes/dev-session.ts

```diff
@@ -233,17 +254,20 @@ async function printWarning(message: string, stdout: Writable) {
   await printLogMessage(outputContent`${outputToken.yellow(message)}`.value, stdout)
 }
 
-async function printError(message: string, stdout: Writable) {
-  await printLogMessage(outputContent`${outputToken.errorText(message)}`.value, stdout)
+async function printError(message: string, stdout: Writable, prefix?: string) {
+  const header = outputToken.errorText(`❌ Error`)
+  const content = outputToken.errorText(`└  ${message}`)
+  await printLogMessage(outputContent`${header}`.value, stdout, prefix)
+  await printLogMessage(outputContent`${content}`.value, stdout, prefix)
 }
 
 async function printSuccess(message: string, stdout: Writable) {
   await printLogMessage(outputContent`${outputToken.green(message)}`.value, stdout)
 }
 
 // Helper function to print to terminal using output context with stripAnsi disabled.
-async function printLogMessage(message: string, stdout: Writable) {
-  await useConcurrentOutputContext({outputPrefix: 'dev-session', stripAnsi: false}, () => {
+async function printLogMessage(message: string, stdout: Writable, prefix?: string) {
```

Comment:
Possible to add a test to cover the overall change (an error tied to an extension should output differently)?